index.html

```diff
@@ -1,200 +1,7 @@
-</ol>
-</li>
-<li>Use the mask texture in the shadow filter to perform a mask on the container that the filter is added to. </li>
-</ol>
-<h2 id="demos-overview"><a class="header-link" href="#demos-overview"></a>Demos overview</h2>
-<ul class="list">
-<li><a href="https://tarvk.github.io/pixi-shadows/build/demos/basic/]">Basic demo</a></li>
-<li><a href="https://tarvk.github.io/pixi-shadows/build/demos/advanced/]">Advanced demo</a></li>
-<li><a href="https://tarvk.github.io/pixi-shadows/build/demos/pixi-lights/]">Pixi-lights demo</a></li>
-<li><a href="https://tarvk.github.io/pixi-shadows/build/demos/system/]">Process demo</a></li>
-</ul>
-<h2 id="using-the-development-environment"><a class="header-link" href="#using-the-development-environment"></a>Using the development environment</h2>
-<h3 id="setup"><a class="header-link" href="#setup"></a>Setup</h3>
-<ul class="list">
-<li>Install <a href="https://nodejs.org/en/">node.js</a></li>
-<li>Run the following command in the root of the project<pre class="hljs"><code>npm <span class="hljs-keyword">install</span></code></pre></li>
-</ul>
-<h3 id="running-demos-(through-which-you-can-test-the-shadows)"><a class="header-link" href="#running-demos-(through-which-you-can-test-the-shadows)"></a>Running demos (through which you can test the shadows)</h3>
-<pre class="hljs"><code>npm <span class="hljs-built_in">run</span> start:[demo <span class="hljs-built_in">name</span>]</code></pre><p>E.G.</p>
-<pre class="hljs"><code>npm <span class="hljs-keyword">run</span><span class="bash"> start:basic</span></code></pre><p>This will start a development server that will live update as you save changes to the source code.</p>
-<h3 id="building-code"><a class="header-link" href="#building-code"></a>Building code</h3>
-<h4 id="build-a-specific-demo"><a class="header-link" href="#build-a-specific-demo"></a>Build a specific demo</h4>
-<pre class="hljs"><code>npm <span class="hljs-built_in">run</span> build-demo:[demo <span class="hljs-built_in">name</span>]</code></pre><p>E.G.</p>
-<pre class="hljs"><code>npm <span class="hljs-keyword">run</span><span class="bash"> build-demo:basic</span></code></pre><h4 id="build-all-demos"><a class="header-link" href="#build-all-demos"></a>Build all demos</h4>
-<pre class="hljs"><code>npm <span class="hljs-keyword">run</span><span class="bash"> build:demos</span></code></pre><h4 id="build-pixi-shadows-itself"><a class="header-link" href="#build-pixi-shadows-itself"></a>Build pixi-shadows itself</h4>
-<pre class="hljs"><code>npm <span class="hljs-keyword">run</span><span class="bash"> build</span></code></pre><h2 id="todo"><a class="header-link" href="#todo"></a>TODO</h2>
-<ul class="list">
-<li>Add more shadow types<ul class="list">
-<li>spotlight shadow</li>
-<li>directional shadow</li>
-</ul>
-</li>
-<li>Improve performance</li>
-</ul>
-    </article>
-  </body>
-</html>
+<H1>Examples index</H1>
+<UL>
+    <LI><A href="./examples/basic/index.html">basic</A></LI>
```

Comment:
Could we use absolute links here maybe? 
Currently it links to the wrong file if you're at the "pixi-shadows/dist" url, but not when you're at the "pixi-shadows/dist/index.html" url. I think the only way for them both to work properly, is to use absolute urls (or at least urls that link from the root of the domain)?

I don't know how smoothly absolute urls would work with vite's dev server vs build, but I assume it should be doable. Otherwise people could get directed to a dead page (currently they may be directed to the source code).

Reply:
the relative links work fine for me on these 4 cases:
http://localhost:4000/pixi-shadows/dist/ , http://localhost:4000/pixi-shadows/dist/index.html (default vscode vite dev server)
http://localhost:8080/pixi-shadows/dist/ , http://localhost:8080/pixi-shadows/dist/index.html (running `npx http-server ..` from project root)

maybe check again? what setup did you use exactly?

Reply:
Ah right, difference is that I didn't have the `/` after dist. Which the static file host I used could deal with, but I don't know if that's standardized or not. But having the trailing `/` is more than good enough in any case.